src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -3,6 +3,7 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using Autodesk.ADPDesktopSDK;
```

Comment:
Where is this namespace coming from?

Reply:
Was coming from `AdpSDKCSharpWrapper.dll`

Reply:
If we are ok with bleeding AdpSDKCSharpWrapper types directly in Dynamo, then makes me wonder why we need an entire Analytics project in between Dynamo and AdpSDKCSharpWrapper. Maybe it made sense when we had multiple analytics providers...but now.
I think it would make it somewhat simpler to consume if we referenced the binary directly in Dynamo and gave up on all the complicated logic of the Analytics.Core. Of course AdpSDKCSharpWrapper is an internal nuget package, so we would have to handle its distribution directly in Dynamo.

Of course if we think we might get into a situation with multiple analytics providers again, then I suggest we keep layer of separation intact 

FYI @QilongTang @mjkkirschner

Reply:
I see what was going on. When consuming the Analytics nuget pacakge, we now have access to the AdpSDKCSharpWrapper.dll at compile time
So any dev writing code could potentially access types directly from that assembly
There are ways to avoid this, like removing the adpsdk binrary from libs, and adding a build.props with a copy target.